### ISSUE TITLE
Configuration for rule "react/jsx-no-bind" is invalid: 	Value {"ignoreRefs":true,"allowArrowFunctions":true,"allowFunctions":false,"allowBind":false,"ignoreDOMComponents":true} should NOT have additional properties.

### ISSUE THREAD

**Issue by Huihuawk** [eslint/eslint#10785]
**Tell us about your environment**

* **ESLint Version:**
version:5.4.0
* **Node Version:**
* **npm Version:**

**What parser (default, Babel-ESLint, etc.) are you using?**
babel-eslint
**Please show your full configuration:**
  "extends": "airbnb",
    "react/jsx-no-bind": ["error", {
      "ignoreRefs": true,
      "allowArrowFunctions": true,
      "allowFunctions": false,
      "allowBind": true,
      "ignoreDOMComponents": true
    }],
<details>
<summary>Configuration</summary>

```js

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js

```

```bash

```

**What did you expect to happen?**

**What actually happened? Please include the actual, raw output from ESLint.**
react.js: 	Configuration for rule "react/jsx-no-bind" is invalid: 	Value {"ignoreRefs":true,"allowArrowFunctions":true,"allowFunctions":false,"allowBind":false,"ignoreDOMComponents":true} should NOT have additional properties.

**Comment by kaicataldo** [comment]
It sounds like you have a misconfiguration for the `react/jsx-no-bind` rule. Can you double check and confirm that it’s not a bug in ESLint?

**Comment by ljharb** [comment]
The `ignoreDOMComponents` option is in v7.11.0+: https://github.com/yannickcr/eslint-plugin-react/commit/5709271977e299bb2142ede92743076175dcce08

This issue should be closed, and should have been filed in eslint-plugin-react in the first place. However, I suspect you just have an old version of eslint-plugin-react.

**Comment by Huihuawk** [comment]
thanks
